src/javascript/JContent/ContentRoute/ToolBar/ContentStatusSelector/ContentStatusSelector.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import {Cloud, Dropdown, Group, Not, Pill, Visibility} from '@jahia/moonstone';
+import JContentConstants from '~/JContent/JContent.constants';
+import {shallowEqual, useDispatch, useSelector} from 'react-redux';
+import {useTranslation} from 'react-i18next';
+import {setActiveContentStatus} from '~/JContent/redux/contentStatus.redux';
+import styles from './ContentStatusSelector.scss';
+
+export const ContentStatusSelector = () => {
+    const {PUBLISHED, PERMISSIONS, VISIBILITY, NO_STATUS} = JContentConstants.statusView;
+    const statusMode = useSelector(state => state.jcontent.contentStatus.active) || NO_STATUS;
+    const contentStatus = useSelector(state => state.jcontent.contentStatus.statusPaths, shallowEqual);
+    const {t} = useTranslation('jcontent');
+    const dispatch = useDispatch();
+
+    const icons = {
```

Comment:
could be outside of component?

Reply:
Will keep this one for now, but I'll make the change next time for content status-related PRs